packages/lesmis/src/utils/getApiUrl.js

```diff
@@ -0,0 +1,31 @@
+/*
+ * Tupaia
+ *  Copyright (c) 2017 - 2021 Beyond Essential Systems Pty Ltd
+ */
+
+export const getApiUrl = () => {
+  // if no env var, use sensible defaults based on the front end url
+  const { hostname } = window.location; // eslint-disable-line no-undef
+
+  // localhost becomes http://localhost:8060
+  if (hostname === 'localhost') {
```

Comment:
Not a major thing, but recently I experience a bug with this kind of logic in tests using the `supertest` framework as it uses `127.0.0.1` instead of `localhost`. Just a heads up in case you wanted to add it here 👍